packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-expandable-answer.tsx

```diff
@@ -0,0 +1,116 @@
+import {
+  buildSmartSnippet,
+  SmartSnippet,
+  SmartSnippetState,
+} from '@coveo/headless';
+import {h, Component, State, Prop, Element, Listen} from '@stencil/core';
+import ArrowDown from '../../images/arrow-down.svg';
+import {
+  InitializeBindings,
+  Bindings,
+  BindStateToController,
+} from '../../utils/initialization-utils';
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'atomic-smart-snippet-expandable-answer',
+  styleUrl: 'atomic-smart-snippet-expandable-answer.pcss',
+  shadow: true,
+})
+export class AtomicSmartSnippetExpandableAnswer {
+  @InitializeBindings() public bindings!: Bindings;
+  public smartSnippet!: SmartSnippet;
+  @BindStateToController('smartSnippet')
+  @State()
+  public smartSnippetState!: SmartSnippetState;
+  public error!: Error;
+  @Element() public host!: HTMLElement;
+
+  /**
+   * The height (in pixels) which, when exceeded by the snippet, displays a "show more" button and truncates the snippet.
+   */
+  @Prop({reflect: true}) minimumSnippetHeightForShowMore = 250;
+  /**
+   * When the answer is partly hidden, how much of its height (in pixels) should be visible.
+   */
+  @Prop({reflect: true}) answerHeightWhenCollapsed = 180;
```

Comment:
In JSUI, those two properties where combined into one, and their default value was `250`, which is pretty big.

I divided them into two properties because I think it would be a bit annoying to press "show more" only to see like 5 extra pixels.

Reply:
I think the prop names are too long and painful to write out for devs.

For `minimumSnippetHeightForShowMore` the jsdoc mentions the maximum height